### PR TITLE
[SPIKE] Feature: Social Login - Web3Auth Wallet Abstraction (Don't merge)

### PR DESCRIPTION
**Pull request by tokdaniel**
Note: E.g Github auth requests permission to some weird app by default, but we can create custom verifiers in the Web3Auth dashboard. For github you need a login provider (Auth0) and the Web3Auth instance needs some setup too, but we can request permission with a custom verifier as Superfluid or e.g B2B Subscriptions Demo, it's configurabe. I have tried, but some config is still missing. Putting this on hold for now.

We need an account for Web3Auth, and in case of custom verifiers Auth0 as well.

EDIT: What's to be done is checking how network switching works after social login. 
B2B Demo doesn't have Wagmi configured, so if you don't use the connect features and open the widget, the connect wallet button does nothing.

### CODE REVIEW

File: examples/b2b-service-demo/src/utils/connectionUtils.ts [hunk 1]
```diff
@@ -3,17 +3,26 @@ import { privateKeyToAccount } from "viem/accounts";
...
+
 const NOOP_ACCOUNT =
   "0x498262a4afde2bdd9e1bef3aafc6442ec36ccf167e527457a211118de93346ba";
 const account = privateKeyToAccount(
   (process.env.NEXT_PUBLIC_THE_THING ?? NOOP_ACCOUNT) as `0x${string}`,
 );
 
+const walletConnectProjectId =
+  process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID ??
+  "952483bf7a0f5ace4c40eb53967f1368";
```
Comment (msoni89) [inline examples/b2b-service-demo/src/utils/connectionUtils.ts:24]: Hardcoded?.

Comment (tokdaniel) [reply examples/b2b-service-demo/src/utils/connectionUtils.ts:24]: yes

File: examples/b2b-service-demo/next.config.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  output: "export",
```
Comment (tokdaniel) [inline examples/b2b-service-demo/next.config.js:4]: this was constantly throwing build errors

**Comment by tokdaniel** [comment]
Closing this for now, we can revisit in the future.

**Pull request closed without merging**